workflow/snakemake_rules/export_for_nextstrain.smk

```diff
@@ -152,15 +152,15 @@ rule auspice_config:
         ## Note: set a value to `None` to exclude it from the produced config JSON
         default_geo_resolution = "division" if build_region in ["north-america", "oceania"] else "country"
         default_map_triplicate = True if build_region in ["reference", "global"] else False
-        if input_set == {"gisaid"}:
+        if "gisaid" in input_set:
             data_provenance = [{"name": "GISAID"}]
             gisaid_clade_coloring = {"key": "GISAID_clade", "title": "GISAID Clade", "type": "categorical"}
             gisaid_epi_isl_coloring = {"key": "gisaid_epi_isl", "type": "categorical"}
             location_coloring = {"key": "location", "title": "Location", "type": "categorical"}
             location_filter = "location"
             originating_lab_filter = "originating_lab"
             submitting_lab_filter  = "submitting_lab"
-        elif input_set == {"open"}:
+        elif "open" in input_set:

```

Comment:
I'd suggest reverting this change as it allows for misuse of this rule to happen silently. Instead, I'd suggest changing the definition of `input_set` further up to specifically exclude a `references` input:

```diff
- input_set = set(config['inputs'])
+ input_set = set(config['inputs']) - {"references"}
```

or alternatively, any input which uses `data/references_sequences.fasta`:

```diff
- input_set = set(config['inputs'])
+ input_set = set(k for k, v in config['inputs'].items() if v.get("sequences") != "data/references_sequences.fasta")
```

Reply:
I applied my first suggestion.